src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -845,22 +846,29 @@ void FancyZones::UpdateDragState(HWND window, require_write_lock) noexcept
         m_dragEnabled = !(shift | mouse);
     }
 
-    const bool windowElevated = IsProcessOfWindowElevated(window);
-    static const bool meElevated = is_process_elevated();
-    static bool warning_shown = false;
-    if (windowElevated && !meElevated)
+    // We should move this logic into the runner once we monitor all elevated processes which are being launched
+    // and notify a user about them, since other modules might care about them as well
+    static std::time_t warning_shown_at = 0;

```

Comment:
Is it safe to initialize this with a zero?

Reply:
yes